include/pisa/type_alias.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <cstdint>
+
+namespace pisa {
+
+using DocId = std::uint32_t;
```

Comment:
Yeah, this is something I'm not sure about. So this one is **not** the same as the underlying type, so if we use the type-safe alias, then this will affect a big chunk of the code base. So I was thinking we can use a regular C++ alias, at least for now.

But I can go back and see what trade-offs this would incur. Honestly, I didn't look into that, so maybe I'll do that. I expect for some significant refactoring to be needed if int and DocId are not interchangable in the type system, but I should check.

Reply:
Let's stick to what we have here for now. We will get back to `type_safe` separately.